src/screens/RegisterAccount.js

```diff
@@ -22,17 +22,20 @@ const RegisterAccount = ({navigation}) => {
             title="Correo electrónico"
             isRequired={IS_REQUIRED}
             password={false}
+            keyboard="email-address"
           />
           <EmailPassTxtInput
             title="Clave"
```

Comment:
Add this on Constants.js

Reply:
@LeyaimMagma, Correct me if this is not true. But, If He adds this to the constant file he will get an error because this is a `prop` from the keyboard